.github/workflows/ci.yml

```
@@ -1898,6 +1900,58 @@ jobs:
           PYTHON_VERSIONS: ${{needs.build-info.outputs.all-python-versions-list-as-string}}
           DEBUG_RESOURCES: ${{ needs.build-info.outputs.debug-resources }}
 
+  build-prod-images-pip:
+    strategy:
+      matrix:
+        python-version: ${{fromJson(needs.build-info.outputs.python-versions)}}
```

```suggestion
        python-version: ${{ fromJson(needs.build-info.outputs.python-versions) }}
```

nitpick: we probably could unify how the spaces is used for `${{ ... }}`